allennlp/data/vocabulary.py

```diff
@@ -316,12 +317,24 @@ def from_files(
         # Parameters
```

Comment:
It's high enough above this that I can't directly make a suggestion to change it, but can you modify the main text of the docstring to say something like `Loads a Vocabulary that was serialized either using save_to_files or inside a model archive file`?

Reply:
Will do